docs/changelog.rst

```diff
@@ -3,11 +3,7 @@
 Changelog
 =========
 
-This is the changelog for *pyirf*.
-
-We use a one-line description of every pull request, roughly in chronological order.
-We don't list every pull request.
-Maintenance and cleanup changes are not of interest to users and are not listed here.
```

Comment:
what was wrong with this information? it is also used in the _gammapy_ changelog

Reply:
Would for simplicity just list every PR, it is more easily extracted from the log, you don't have to judge
which ones are important and which are not.


Also: I find it demeaning to say "this is not of interest to users".
Users should be able to decide themselves what is interesting to them.